.circleci/config.yml

```diff
@@ -98,7 +99,7 @@ jobs:
 
```

Comment:
Would it be possible to unify the other Dockerized deployment steps as well? I see `ci/deb-ppa-upload/ci-deploy.sh` as a potential candidate

Reply:
Although there's a slight difference there (at least wrt. one vs two `docker-compose run` calls), so probably some extra refactors will be needed, leaving that up to you